apps/meteor/packages/rocketchat-i18n/i18n/en.i18n.json

```
@@ -3075,7 +3075,9 @@
   "Livechat_agents": "Omnichannel agents",
   "Livechat_Agents": "Agents",
   "Livechat_allow_manual_on_hold": "Allow agents to manually place chat On Hold",
-  "Livechat_allow_manual_on_hold_Description": "If enabled, the agent will get a new option to place a chat On Hold, provided the agent has sent the last message",
+  "Livechat_allow_manual_on_hold_Description": "If enabled, the agent will get the option to place a chat On Hold",
+  "Livechat_allow_manual_on_hold_upon_agent_engagement_only": "Allow agents to manually place chat On Hold, provided the agent has sent the last message",
+  "Livechat_allow_manual_on_hold_upon_agent_engagement_only_Description": "If enabled, the agent will get a new option to place a chat On Hold, provided the agent has sent the last message, otherwise agents are able to place chats on Hold irrespective",
```

```suggestion
  "Livechat_allow_manual_on_hold_upon_agent_engagement_only_Description": "If enabled, the agent will get a new option to place a chat On Hold, provided the agent has sent the last message, otherwise agents are able to place chats On Hold irrespective",
```

Let's wait for arthur's check, but maybe too much "on hold" 🙈

---

This is now resolved. New translations are in the latest push